### PR TITLE
Version 0.3.0

### PR DESCRIPTION
**Pull request by nkern**
This bumps the version to 0.3.0, which as of Jan 2021 has all of the necessary plumbing for the H1C IDR2.2 upper limit analysis. Once this is merged I will tag an associated release.

Major developments over 0.2.0 include Tan et al. 2021 errorbar methods, in addition to spherical binning, and other minor features.

@philbull and @acliu let me know if there is anything you'd like to see get into hera_pspec before we tag this release. thanks!

### CODE REVIEW

**Comment by philbull** [comment]
Lots of docstring changes were needed to get RTD to build. We should probably include RTD builds in the CI...

File: hera_pspec/utils.py [hunk 1]
```diff
@@ -1393,10 +1399,47 @@ def uvp_noise_error(uvp, auto_Tsys, err_type='P_N', precomp_P_N=None):
...
+def apply_P_SN_correction(uvp, P_SN='P_SN', P_N='P_N'):
+    """
+    Apply correction factor to P_SN errorbar in stats_array to account
+    for double counting of noise by using data as proxy for signal.
+    See Jianrong Tan et al. 2021 (Errorbar methodologies).
+    Operates in place. Must have both P_SN and P_N in stats_array.
+
+    Args:
+        uvp : UVPSpec object
+            With P_SN errorbar (not variance) as a key
```
Comment (JianrongTan) [inline hera_pspec/utils.py:1428]: "with P_SN and P_N"

File: hera_pspec/utils.py [hunk 2]
```diff
@@ -1393,10 +1399,47 @@ def uvp_noise_error(uvp, auto_Tsys, err_type='P_N', precomp_P_N=None):
...
+        P_SN : str
+            Key in stats array for P_SN errorbar
```
Comment (JianrongTan) [inline hera_pspec/utils.py:1430]: Miss the "_"

File: hera_pspec/utils.py (hunk 1 above)
Comment (nkern) [reply hera_pspec/utils.py:1428]: yup, thanks

File: hera_pspec/utils.py (hunk 2 above)
Comment (nkern) [reply hera_pspec/utils.py:1430]: yup, thanks

**Pull request merged**